src/models/v2/components.ts

```diff
@@ -76,9 +76,7 @@ export class Components extends BaseModel<v2.ComponentsObject> implements Compon
   }
 
   operations(): OperationsInterface {
-    const operations: OperationInterface[] = [];
-    this.channels().forEach(channel => operations.push(...channel.operations().all()));
-    return new Operations(operations);
+    return new Operations([]);
```

Comment:
We don't have `operations` in `components` in v2, so we should return empty collection.